samples/README.md

```diff
@@ -19,23 +19,23 @@ Sample Location: [peoplenet_detector](./peoplenet_detector)
 
 Preview:
 
-![](peoplenet-blur-demo-loop-400.webp)
+![](peoplenet_detector/assets/peoplenet-blur-demo-loop-400.webp)
 
 ### Car Detection and Classification (Nvidia detectors and classifiers, Nvidia tracker)
 
 Sample Location: [nvidia_car_classification](./nvidia_car_classification)
 
 Preview:
 
-![](nvidia-car-classification-loop-400.webp)
+![](nvidia_car_classification/assets/nvidia-car-classification-loop-400.webp)
 
 ### Line crossing demo
 
 Sample Location: [line_crossing](./line_crossing)
 
 Preview:
 
-![](line-crossing-loop-400.webp)
+![](line_crossing/assets/line-crossing-loop-400.webp)
```

Comment:
@denisvmedyantsev in the ticket, there is a note to rename it.

Reply:
fixed